### PR TITLE
data: IPratikMali

### PR DESCRIPTION
**Pull request by IPratikMali**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4998"></a>

### CODE REVIEW

File: data/IPratikMali.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Pratik Mali",
+  "displayStatsPublic" : true,
```
Comment (swagataroy30) [inline data/IPratikMali.json:3]:
```suggestion
```

File: data/IPratikMali.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "type": "personal",
+  "bio": "Aspiring App-Dev",
+  "avatar" : "https://github.com/IPratikMali.png",
```
Comment (swagataroy30) [inline data/IPratikMali.json:6]:
```suggestion
```

**Review by swagataroy30 — COMMENTED** [review]
In the recent update, we don't need to add avatar: anymore, it will auto-pick your GitHub profile picture and also `"displayStatsPublic" : true` is no longer required. So, please remove that.
Updated docs: https://linkfree.eddiehub.io/docs/quickstart
Thank You!

File: data/IPratikMali.json (hunk 2 above)
Comment (IPratikMali) [reply data/IPratikMali.json:6]: okay , appreciate your help

**Review by sidharthviz — APPROVED** [review]
Looks Good now 🎉

**Review by mahfuztbt — APPROVED** [review]

**Review by samyabrata-maji — APPROVED** [review]

**Review by swagataroy30 — APPROVED** [review]
LGTM!

**Comment by sansarmaske** [comment]
LGTM 👍

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**